aerogear/mobile-security-service
# fix(ui): Rename component (AEROGEAR-8875)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8875

## What

Renamed component from `AppOverviewContainer` to `AppOverview`.

## Why

Since it is not a container it should not have been called this.

## How
Add an short answer for: How it was done? (E.g By removing this feature from ... OR By removing just the button but not its implementation ... ) 

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Verify that the component name and filename has been changed.
2. Verify that the Mobile App Detailed page still displays. 

## Checklist:


## Progress
